### PR TITLE
Update instructions to build on CADES SHPC Condo and allow config with Boost 1.61

### PR DESCRIPTION
**Pull request by dalg24**

### CODE REVIEW

File: packages/Meshfree/test/CMakeLists.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   FAIL_REGULAR_EXPRESSION "data race;leak;runtime error"
   )
 
-IF (HAVE_DTK_BOOST)
+IF (HAVE_DTK_BOOST AND ((BOOST_VERSION VERSION_EQUAL 1.62.0) OR (BOOST_VERSION VERSION_GREATER 1.62.0)))
```
Comment (sslattery) [inline packages/Meshfree/test/CMakeLists.txt]: So this test only works with version 1.62 or greater?

File: scripts/cades_shpc_condo_cmake [hunk 2]
```diff
@@ -11,12 +9,12 @@ ARGS=(
     -D CMAKE_BUILD_TYPE=Debug
     -D BUILD_SHARED_LIBS=ON
     ### COMPILERS AND FLAGS ###
-    -D CMAKE_CXX_FLAGS="-g -arch=${GPU_ARCH} -lineinfo \
```
Comment (sslattery) [inline scripts/cades_shpc_condo_cmake]: so now extended lambda only gets passed to cxx11 flags? What exactly does extended lambda do again?

Comment (dalg24) [reply scripts/cades_shpc_condo_cmake]: See  #353

File: packages/Meshfree/test/CMakeLists.txt (hunk 1 above)
Comment (dalg24) [reply packages/Meshfree/test/CMakeLists.txt]: See #339

File: scripts/cades_shpc_condo_cmake (hunk 2 above)
Comment (dalg24) [reply scripts/cades_shpc_condo_cmake]: https://devblogs.nvidia.com/new-compiler-features-cuda-8/

**Pull request merged**